deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml

```diff
@@ -63,8 +63,43 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: egov-idgen
-  - name: EGOV_WORKFLOW_HOST
-    valueFrom:
-      configMapKeyRef:
-        name: egov-service-host
-        key: egov-workflow-v2
+  - name: ATTENDANCE_SERVICE_LOG_KAFKA_CREATE_TOPIC
+    value: "save-attendance-log"
+  - name: ATTENDANCE_SERVICE_LOG_KAFKA_UPDATE_TOPIC
+    value: "update-attendance-log"
+  - name: ATTENDANCE_SERVICE_REGISTER_KAFKA_CREATE_TOPIC
+    value: "save-attendance"
+  - name: ATTENDANCE_SERVICE_REGISTER_KAFKA_UPDATE_TOPIC
+    value: "update-attendance"
+  - name: ATTENDANCE_SERVICE_STAFF_KAFKA_CREATE_TOPIC
+    value: "save-staff"
+  - name: ATTENDANCE_SERVICE_STAFF_KAFKA_UPDATE_TOPIC
+    value: "update-staff"
+  - name: ATTENDANCE_SERVICE_ATTENDEE_KAFKA_CREATE_TOPIC
+    value: "save-attendee"
+  - name: ATTENDANCE_SERVICE_ATTENDEE_KAFKA_UPDATE_TOPIC
+    value: "update-attendee"
+  {{- if index .Values "attendance-service-register-default-offset" }}
+  - name: ATTENDANCE_SERVICE_REGISTER_DEFAULT_OFFSET
+    value: {{ index .Values "attendance-service-register-default-offset" | quote }} 
+  {{- end }}
+  {{- if index .Values "attendance-service-register-default-limit" }}
+  - name: ATTENDANCE_SERVICE_REGISTER_DEFAULT_LIMIT
+    value: {{ index .Values "attendance-service-register-default-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "attendance-service-register-search-max-limit" }}
+  - name: ATTENDANCE_SERVICE_REGISTER_SEARCH_MAX_LIMIT
+    value: {{ index .Values "attendance-service-register-search-max-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "attendance-service-individual-service-integration-required" }}
+  - name: ATTENDANCE_SERVICE_INDIVIDUAL_SERVICE_INTEGRATION_REQUIRED
```

Comment:
What does this do? Just a flag to turn on/off, I assume?

Reply:
Staff / Individual / Document service integration is configuration driven. Will integrate them once services will be available.